src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -191,7 +192,14 @@ export default class ClinicalData extends React.Component<
 
     private tableDataStore = new ClinicalDataEnrichmentStore(
         () => {
-            return this.props.store.clinicalDataEnrichmentsWithQValues.result;
+            // exclude survival months and status from the data store
+            return this.props.store.clinicalDataEnrichmentsWithQValues.result.filter(
+                d =>
+                    !getSurvivalAttributeIdsFromPrefix(
```

Comment:
this is not really optimal.  you are recreating this array of attributeIds every cycle, when i think it will always tbe the same.   also, using the "in" operator would be faster.     so, make an object { '??_SURVIVAL':null,  ...  } and then condition:     
```d.clincialAttribute.clinicalAttributeId in obj```

Reply:
updated